OADP-3170: Release Notes for OADP 1.1.8 (#75824)
openshift/openshift-docs: modules/oadp-release-notes-1-1-8.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-release-notes-1-1.adoc

:_mod-docs-content-type: REFERENCE
[id="migration-oadp-release-notes-1-1-8_{context}"]
= {oadp-short} 1.1.8 release notes

The {oadp-first} 1.1.8 release notes lists any known issues. There are no resolved issues in this release.

> [id="known-issues1.1.8_{context}"]

Author: ```suggestion
[id="known-issues1-1-8_{context}"]
```

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-release-notes-1-1.adoc

:_mod-docs-content-type: REFERENCE
[id="migration-oadp-release-notes-1-1-8_{context}"]
= {oadp-short} 1.1.8 release notes

> The {oadp-first} 1.1.8 release notes lists any resolved issues and known issues.

Author: ```suggestion
The {oadp-first} 1.1.8 release notes lists any known issues. There are no resolved issues in this release.
```

> [id="resolved-issues1.1.8_{context}"]

Reviewer: I'd keep your ID format consistent `1-1-8` same as KIs ID.

> [id="resolved-issues1.1.8_{context}"]

Author: ```suggestion
[id="resolved-issues1-1-8_{context}"]
```

== Resolved issues

There are no resolved issues in {oadp-short} 1.1.8.

[id="known-issues1.1.8_{context}"]
== Known issues

> For a complete list of all known issues in in {oadp-short} 1.1.8, see the list of link:https://issues.redhat.com/issues/?filter=12435971[OADP 1.1.8 known issues] in Jira.

Reviewer: ```suggestion
For a complete list of all known issues in {oadp-short} 1.1.8, see the list of link:https://issues.redhat.com/issues/?filter=12435971[OADP 1.1.8 known issues] in Jira.
```

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-release-notes-1-1.adoc

:_mod-docs-content-type: REFERENCE
[id="migration-oadp-release-notes-1-1-8_{context}"]
= OADP 1.1.8 release notes

The OADP 1.1.8 release notes lists any resolved issues and known issues.

[id="resolved-issues1.1.8_{context}"]
== Resolved issues

The following highlighted issues are resolved in OADP 1.1.8:

.CVE-2023-45288: GoLang: `net/http`, `x/net/http2`: close connections when receiving too many headers

> A security vulnerability was discovered with the implementation of the HTTP/2 protocol in the Go programming language. There were insufficient restrictions on the quantity of CONTINUATION frames that could be transmitted within a single stream. An attacker could exploit this to potentially cause a Denial-of-Service (DoS) attack.

Author: ```suggestion
A security vulnerability was discovered with the implementation of the HTTP/2 protocol in the Go programming language. There were insufficient restrictions on the quantity of CONTINUATION frames that could be transmitted within a single stream. An attacker could exploit this to potentially cause a denial-of-service (DoS) attack.
```

//
// * backup_and_restore/oadp-release-notes-1-1.adoc

:_mod-docs-content-type: REFERENCE
[id="migration-oadp-release-notes-1-1-8_{context}"]
= {oadp-short} 1.1.8 release notes

The {oadp-first} 1.1.8 release notes lists any resolved issues and known issues.

[id="resolved-issues1.1.8_{context}"]
== Resolved issues

The following highlighted issues are resolved in {oadp-short} 1.1.8:

.CVE-2023-45288: GoLang: `net/http`, `x/net/http2`: close connections when receiving too many headers

A security vulnerability was discovered with the implementation of the HTTP/2 protocol in the Go programming language. There were insufficient restrictions on the quantity of CONTINUATION frames that could be transmitted within a single stream. An attacker could exploit this to potentially cause a denial-of-service (DoS) attack.

For more information, see link:https://access.redhat.com/security/cve/cve-2023-45288[(CVE-2023-45288)]

For a complete list of all issues resolved in the release of OADP 1.1.8, see the list of link:https://issues.redhat.com/issues/?filter=12435970[OADP 1.1.8 resolved issues] in Jira.

[id="known-issues1.1.8_{context}"]
== Known issues

> // There are no known issues in the release of {oadp-short} 1.1.8.

Author: ```suggestion
There are no known issues in {oadp-short} 1.1.8.
```

:_mod-docs-content-type: REFERENCE
[id="migration-oadp-release-notes-1-1-8_{context}"]
= {oadp-short} 1.1.8 release notes

The {oadp-first} 1.1.8 release notes lists any resolved issues and known issues.

[id="resolved-issues1.1.8_{context}"]
== Resolved issues

The following highlighted issues are resolved in {oadp-short} 1.1.8:

.CVE-2023-45288: GoLang: `net/http`, `x/net/http2`: close connections when receiving too many headers

A security vulnerability was discovered with the implementation of the HTTP/2 protocol in the Go programming language. There were insufficient restrictions on the quantity of CONTINUATION frames that could be transmitted within a single stream. An attacker could exploit this to potentially cause a denial-of-service (DoS) attack.

For more information, see link:https://access.redhat.com/security/cve/cve-2023-45288[(CVE-2023-45288)]

For a complete list of all issues resolved in the release of OADP 1.1.8, see the list of link:https://issues.redhat.com/issues/?filter=12435970[OADP 1.1.8 resolved issues] in Jira.

[id="known-issues1.1.8_{context}"]
== Known issues

// There are no known issues in the release of {oadp-short} 1.1.8.

> // The {oadp-short} 1.1.8 release has the following known issue:

Author: ```suggestion
The {oadp-short} 1.1.8 release has the following known issue:

.Additional secrets not mounted in all providers

link:https://issues.redhat.com/browse/OADP-3971[(OADP-3971)]
```

:_mod-docs-content-type: REFERENCE
[id="migration-oadp-release-notes-1-1-8_{context}"]
= {oadp-short} 1.1.8 release notes

The {oadp-first} 1.1.8 release notes lists any resolved issues and known issues.

[id="resolved-issues1.1.8_{context}"]
== Resolved issues

The following highlighted issues are resolved in {oadp-short} 1.1.8:

.CVE-2023-45288: GoLang: `net/http`, `x/net/http2`: close connections when receiving too many headers

A security vulnerability was discovered with the implementation of the HTTP/2 protocol in the Go programming language. There were insufficient restrictions on the quantity of CONTINUATION frames that could be transmitted within a single stream. An attacker could exploit this to potentially cause a denial-of-service (DoS) attack.

For more information, see link:https://access.redhat.com/security/cve/cve-2023-45288[(CVE-2023-45288)]

For a complete list of all issues resolved in the release of OADP 1.1.8, see the list of link:https://issues.redhat.com/issues/?filter=12435970[OADP 1.1.8 resolved issues] in Jira.

[id="known-issues1.1.8_{context}"]
== Known issues

// There are no known issues in the release of {oadp-short} 1.1.8.

> The {oadp-short} 1.1.8 release has the following known issue:
> 
> .Additional secrets not mounted in all providers
> 
> link:https://issues.redhat.com/browse/OADP-3971[(OADP-3971)]

Author: ```suggestion
```
